.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true
+  },
+  extends: [
+    'plugin:react/recommended',
+    'standard-with-typescript'
+  ],
+  overrides: [
+  ],
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module'
+  },
+  plugins: [
+    'react'
+  ],
+  rules: {
+  }
```

Comment:
You should add some rules may looks like below:
```
"rules": {
    "react/prop-types": 0,
    "indent": ["error", 2],
    "linebreak-style": 1,
    "quotes": ["error", "double"]
},
```
And more read here: https://www.geeksforgeeks.org/how-to-configure-eslint-for-react-projects/

Reply:
I have added eslint rules and fixed some easy issues.